test/EventStream-e2e-browser.html

```diff
@@ -0,0 +1,49 @@
+<!--
+End-to-end test program for the event stream in the browser
+
+Steps:
+- npm run compile
+- npm run build
+- Replace the path to particle.min.js
+- Replace your access token in `auth`
```

Comment:
It would be nice to be able to pass these variables in a better way, but I don't want perfect to be the enemy of good enough here...